internal/repository/packer_manager.go

```diff
@@ -36,7 +36,7 @@ type packerManager struct {
 	packers []*Packer
 }
 
-const minPackSize = 4 * 1024 * 1024
+const defaultMinPackSize = 4 * 1024 * 1024
```

Comment:
maybe move to `packer_manager_test.go` as this is now only used there?

Reply:
Agreed, probably best.